docs/pages/access-controls/access-request-plugins/opsgenie.mdx

```diff
@@ -75,10 +75,12 @@ spec:
         - approve: 1
           deny: 1
       annotations:
-        teleport.dev/schedules: ['teleport-access-request-notifications']
+        teleport.dev/notify-services: ['teleport-access-request-notifications']
 ```
 
-The `teleport.dev/schedules` annotation specifies the schedule the alert will be be created for.
+The `teleport.dev/notify-services` annotation specifies the schedules the alert will be be created for.
+The `teleport.dev/schedules` annotation specifies the schedules the alert will check, and auto approve the
+Access Request if the requesting user is on-call.
```

Comment:
Should we also include `teleport.dev/schedules` in the spec above? It's a bit confusing when the example shows only one annotation and the text after it discusses two annotations.